### ISSUE TITLE
Backintime KDE as root with ssh / key based login w/o password

### ISSUE THREAD

**Issue by Germar** [bit-team/backintime#47]
Hi,

thank you for this great tool! I'm used to use backintime with ssh and key based authorization, with no password set on the key.

This does work still for a non-root account, but for a root account it does not work as the setup tries to save the password.

However, with an empty password there is no need to save, nor to cache etc. a password.

What I would recommend is to add a check-box 'key with no password' or just don't try to save when the password string is empty.

The reason I set up systems this way and not as a user account is that at the moment I set up the system there is no other user then root, the owner of the system set up his own account.

Having backintime backing up /home it will automatically catch the user account plus is able to save /etc, /boot, /var/log - which enables me to help in case something happens.

Any ideas how to get this running as root the described way? Or do I have to change the code?

greetings kai

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2014-02-04T21:53:19Z
 * owner: germar
 * assignee: germar
 * the launchpad url was https://bugs.launchpad.net/bugs/1276348

**Comment by Germar** [comment]
(by germar)
Please try out attached patch with 'sudo patch /usr/bin/backintime-kde4 < backintime-kde4.diff'

**Comment by Germar** [comment]
(by sparhawkthesecond)
This fix seems good to me.

I use Arch, so I had to modify things slightly. The patch failed for me, since Arch uses python 3 by default. Hence, the patched target has the post-patch line as `python2`, not `python`. I could manually patch the file though. Also, I had to create the fuse group (it doesn't exist in Arch) and add myself to it.

sudo groupadd fuse
sudo usermod -aG fuse root

Thanks for the fix!
